File: app/src/main/java/com/namseox/adv/SplashActivity.java
package com.namseox.adv;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.lvt.ads.callback.AdCallback;
import com.lvt.ads.util.AppOpenManager;

public class SplashActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        AdCallback adCallback;
        adCallback = new AdCallback(){
            @Override
            public void onNextAction() {
                super.onNextAction();
                startActivity(new Intent(SplashActivity.this,MainActivity.class));
            }
        };
        AppOpenManager.getInstance().loadOpenAppAdSplash(this,"ca-app-pub-3940256099942544/3419835294",3000,10000,true,adCallback);
        AppOpenManager.getInstance().onCheckShowSplashWhenFail(this,adCallback,1000);
    }
}

File: theme_call/ST003_CallTheme-ngay1/app/src/main/java/com/namseox/st003_calltheme/ui/splash/SplashActivity.kt
package com.namseox.st003_calltheme.ui.splash

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import android.os.PersistableBundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import androidx.databinding.ViewDataBinding
import com.namseox.st003_calltheme.MainActivity
import com.namseox.st003_calltheme.R
import com.namseox.st003_calltheme.ui.tutorial.TutorialActivity
import com.namseox.st003_calltheme.utils.Const
import com.namseox.st003_calltheme.utils.SharedPreferenceUtils
import com.namseox.st003_calltheme.utils.Utils.setDefaultScreen
import com.namseox.st003_calltheme.utils.Utils.setFullScreen

class SplashActivity : AppCompatActivity() {
    private val handler = Handler(Looper.myLooper()!!)
    private val runnable = Runnable {
        if(SharedPreferenceUtils.getInstance(applicationContext).getBooleanValue(Const.CHECKTUTORIAL)){
            startActivity(Intent(this, MainActivity::class.java))
        }else{
            startActivity(Intent(this, TutorialActivity::class.java))
        }
    }
override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_splash)
}

    override fun onStart() {
        super.onStart()
        handler.postDelayed(runnable, 1500)
    }

    override fun onPause() {
        super.onPause()
        handler.removeCallbacks(runnable)
    }

    override fun onDestroy() {
        super.onDestroy()

    }
}